File: dev/src/EleveBundle/Entity/Classe.php
<?php

namespace EleveBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Classe
 *
 * @ORM\Table(name="classe")
 * @ORM\Entity(repositoryClass="EleveBundle\Repository\ClasseRepository")
 */
class Classe
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="nom_classe", type="string", length=255)
     */
    private $nomClasse;

    /**
     * @var int
     *
     * @ORM\Column(name="nbre_eleve", type="integer")
     */
    private $nbreEleve;

    /**
     * @var int
     *
     * @ORM\Column(name="capacite", type="integer")
     */
    private $capacite;

    /**
     * @var string
     *
     * @ORM\Column(name="salle", type="string", length=255)
     */
    private $salle;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set nomClasse
     *
     * @param string $nomClasse
     *
     * @return Classe
     */
    public function setNomClasse($nomClasse)
    {
        $this->nomClasse = $nomClasse;

        return $this;
    }

    /**
     * Get nomClasse
     *
     * @return string
     */
    public function getNomClasse()
    {
        return $this->nomClasse;
    }

    /**
     * Set nbreEleve
     *
     * @param integer $nbreEleve
     *
     * @return Classe
     */
    public function setNbreEleve($nbreEleve)
    {
        $this->nbreEleve = $nbreEleve;

        return $this;
    }

    /**
     * Get nbreEleve
     *
     * @return int
     */
    public function getNbreEleve()
    {
        return $this->nbreEleve;
    }

    /**
     * Set capacite
     *
     * @param integer $capacite
     *
     * @return Classe
     */
    public function setCapacite($capacite)
    {
        $this->capacite = $capacite;

        return $this;
    }

    /**
     * Get capacite
     *
     * @return int
     */
    public function getCapacite()
    {
        return $this->capacite;
    }

    /**
     * Set salle
     *
     * @param string $salle
     *
     * @return Classe
     */
    public function setSalle($salle)
    {
        $this->salle = $salle;

        return $this;
    }

    /**
     * Get salle
     *
     * @return string
     */
    public function getSalle()
    {
        return $this->salle;
    }
}

File: Java/gestioneleve/src/entities/classe.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

/**
 *
 * @author wejdene
 */
public class classe {
   private int id ;
   private String nom_classe;
   private int nbre_eleve;
   private int capacite;
   private String salle;

    public classe(int id, String nom_classe, int nbre_eleve, int capacite, String salle) {
        this.id = id;
        this.nom_classe = nom_classe;
        this.nbre_eleve = nbre_eleve;
        this.capacite = capacite;
        this.salle = salle;
    }

    public classe() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom_classe() {
        return nom_classe;
    }

    public void setNom_classe(String nom_classe) {
        this.nom_classe = nom_classe;
    }

    public int getNbre_eleve() {
        return nbre_eleve;
    }

    public void setNbre_eleve(int nbre_eleve) {
        this.nbre_eleve = nbre_eleve;
    }

    public int getCapacite() {
        return capacite;
    }

    public void setCapacite(int capacite) {
        this.capacite = capacite;
    }

    public String getSalle() {
        return salle;
    }

    public void setSalle(String salle) {
        this.salle = salle;
    }

    @Override
    public String toString() {
        return "classe{" + "id=" + id + ", nom_classe=" + nom_classe + ", nbre_eleve=" + nbre_eleve + ", capacite=" + capacite + ", salle=" + salle + '}';
    }
    
}
